packages/agent-nodejs-testing/package.json

```diff
@@ -28,18 +28,18 @@
     "superagent": "^8.1.2"
   },
   "devDependencies": {
-    "@forestadmin/agent": "1.41.12",
-    "@forestadmin/datasource-customizer": "1.47.2",
-    "@forestadmin/datasource-sql": "1.14.18",
-    "@forestadmin/datasource-toolkit": "^1.34.5",
-    "@forestadmin/forestadmin-client": "^1.27.8",
-    "sequelize": "6.35.2",
-    "sqlite3": "5.1.6"
+    "@forestadmin/agent": "1.52.0",
+    "@forestadmin/datasource-customizer": "1.55.2",
+    "@forestadmin/datasource-sql": "1.14.31",
+    "@forestadmin/datasource-toolkit": "1.43.0",
+    "@forestadmin/forestadmin-client": "1.34.2",
+    "sequelize": "6.37.5",
+    "sqlite3": "^5.1.7"
   },
   "peerDependencies": {
-    "@forestadmin/agent": ">=1.41.1",
-    "@forestadmin/datasource-customizer": ">=1.47.0",
-    "@forestadmin/datasource-toolkit": ">=1.34.0",
-    "@forestadmin/forestadmin-client": ">=1.27.8"
+    "@forestadmin/agent": ">=1.52.0",
```

Comment:
Not really sure of the version where we unlock the form layout feature, that why I have put the last one.